behave/configuration.py

```diff
@@ -293,6 +293,14 @@ class ConfigError(Exception):
      dict(action="store_true",
           help="Stop running tests at the first failure.")),
 
+    (("--fork",),
+     dict(action="store_true",
+          help="Run each feature in a different process")),
+
+    (("--timeout",),
+     dict(type=int,
+          help="Set a max timeout for eachs")),
```

Comment:
for eachs -> for each feature
I would also say that it only works in combination with fork

Reply:
Done before merging.